extension.js

```diff
@@ -22,6 +22,7 @@ const Extension = imports.misc.extensionUtils.getCurrentExtension();
 const MenuItems = Extension.imports.menu_items;
 const Promise = Extension.imports.promise.Promise;
```

Comment:
Why? And how? This is not API behaviour but extension logic. Doing something once these specific api requests are done.